modules/quirks/qualcomm/sdm845-modem.nix

```diff
@@ -29,13 +60,16 @@ in
         , firmwareFilesList
         }:
 
-        runCommand "sdm845-uncompressed-firmware-share" {
+        runCommand "qcom-modem-uncompressed-firmware-share" {
           firmwareFiles = buildEnv {
-            name = "sdm845-uncompressed-firmware";
+            name = "qcom-modem-uncompressed-firmware";
             paths = firmwareFilesList;
             pathsToLink = [
-              "/lib/firmware/qcom/sdm845"
-            ];
+              "/lib/firmware/rmtfs"
+            ]
+              ++ optional cfg.sdm845-modem.enable "/lib/firmware/qcom/sdm845"
+              ++ optional cfg.sc7180-modem.enable "/lib/firmware/qcom/sc7180-trogdor"
```

Comment:
This might not be entirely *correct*, but is close enough.

We may want to make this configurable down the line if/when more 7c devices ends-up in here.